web/src/routes/OrganizationsList/Organization/Organization.tsx

```diff
@@ -114,7 +119,7 @@ export default function Organization() {
           isUser={isUserOrganization}
         />
       ),
-      visible: true,
+      visible: fetchTabVisibility('Robot accounts'),
```

Comment:
Is there a reason why you are changing this?

Reply:
This should only be available for specific permissions